proto/substrait/algebra.proto

```diff
@@ -437,7 +437,10 @@ message Rel {
     ExtensionMultiRel extension_multi = 10;
     ExtensionLeafRel extension_leaf = 11;
     CrossRel cross = 12;
-    //Physical relations
+    ReferenceRel reference = 21;
```

Comment:
This starts the [// Write relations](url) section presumably?

Reply:
I actually added `// Write relations` but then I thought "Oh wait, RefRel could be used in a query too".

Then I thought "Are write (which does not affect the catalog) and Ddl (which does affect the catalog) really in the same category?"

Then I despaired and remembered Jacques advice that trying to categorize these things was doomed and I erased the comment.

These are all just "not physical relations" :laughing: .  On the web page they are currently all listed under "logical relations".  If we want to take a stab at further categorization I'd like to do that in a follow-up.